engine.py

```diff
@@ -914,9 +915,22 @@ def __setup_sio(self, hostname, port, namespace):
         :rtype: bool
         """
 
+        # Set up kwargs to pass to socketio Client
+        client_kwargs = {}
```

Comment:
Minor suggestion to group all `ShotGridAliasSocketIoClient` named parameters together

```suggestion
        client_kwargs = {
          "timeout": 60 * 3,
        }
```